plugin/pkg/auth/authenticator/token/bootstrap/bootstrap.go

```diff
@@ -180,8 +184,10 @@ func isSecretExpired(secret *api.Secret) bool {
 // Copied from kubernetes/cmd/kubeadm/app/util/token
 
 var (
-	tokenRegexpString = "^([a-z0-9]{6})\\.([a-z0-9]{16})$"
-	tokenRegexp       = regexp.MustCompile(tokenRegexpString)
+	// TokenRegexpString defines id.secret regular expression pattern
+	TokenRegexpString = "^([a-z0-9]{6})\\.([a-z0-9]{16})$"
+	// TokenRegexp is a compiled regular expression of TokenRegexpString
+	TokenRegexp = regexp.MustCompile(TokenRegexpString)
```

Comment:
don't export these, we don't really want things starting to depend on this package for these

Reply:
sorry for my mistake, thank you!